jira_fetcher.go

```diff
@@ -158,6 +159,16 @@ loop:
 	for off := ev.Offset; off < ev.Offset+totalSize; off += pageSize {
 		ctx := ctx
 		ctx = updateContextWithPage(ctx, off)
+
+		progressCurrentPage := int64(off / pageSize)
+		progressTotalPages := int64(off/pageSize) + pagesAmount
+
+		if off == 0 {
+			if err := f.createProgress(ctx, ev, 0, pagesAmount); err != nil {
```

Comment:
```suggestion
			if err := f.createProgress(ctx, ev, progressCurrentPage, progressTotalPages); err != nil {
```
Just in case we run with `force = true` and `page = 1`. Same in other places.

Reply:
Done.